src/metabase/task/sync.clj

```diff
@@ -0,0 +1,117 @@
+(ns metabase.task.sync
+  (:require [clojure.tools.logging :as log]
+            [clojurewerkz.quartzite
+             [conversion :as qc]
+             [jobs :as jobs]
+             [triggers :as triggers]]
+            [clojurewerkz.quartzite.schedule.cron :as cron]
+            [metabase
+             [cache-database :as cache-database]
+             [task :as task]
+             [util :as u]]
+            [metabase.models.database :refer [Database]]
+            [metabase.sync-database
+             [analyze :as analyze]
+             [cached-values :as cached-values]
+             [classify :as classify]]
+            [toucan.db :as db]))
+
+(def ^:private ^:const classify-databases-job-key     "metabase.task.%s-databases.job-%s")
+(def ^:private ^:const classify-databases-trigger-key "metabase.task.%s-databases.trigger-%s")
+
+(defonce ^:private classify-databases-job (atom nil))
+(defonce ^:private classify-databases-trigger (atom nil))
```

Comment:
This PR is becoming increasingly scary to me as I realize there are all sorts of things like this var here that are defined but never used *anywhere*. It's hard to tell whether it's leftover code that should have been removed or whether some of the other code is written incorrectly because and should be using this but isn't

I'm worried that if we merge this for 25 it will break something and I'll be lost as to how to fix it because I can't understand what's supposed to be happening and what's not working right